### ISSUE TITLE
add a kmip server

### ISSUE THREAD

**Issue by emisca** [HenryGP/om_ansible#22]
ticket to track the effort to add a kmip server.

**Comment by emisca** [comment]
trying to add it to the ldapserver container:

```
apt-get install python-dev libffi-dev libssl-dev libsqlite3-dev python-pip
pip install pykmip
```

then we need to generate a set of ssl certs and create the conf file... WIP!

**Comment by HenryGP** [comment]
This has been resolved in Pull Requests #37 and #36
